tests/test_models/deterministic.py

```diff
@@ -171,35 +171,34 @@ def absorbing_retirement_filter(retirement, lagged_retirement):
 
 
 @dataclass
-class DiscreteConsumptionStatus:
-    low: int = 1
-    high: int = 2
-
-
-DiscreteWealthStatus = make_dataclass(
-    "DiscreteWealthStatus", [(f"level_{w}", int, w) for w in range(1, 401)]
-)
+class ConsumptionStatus:
```

Comment:
```suggestion
class ConsumptionChoice:
```
Status sounds really weird here. Apologies for not noting before. I think `Working` is a special case in the sense that a choice turning directly into a state can be better seen as a `Status`. 

More importantly: Define this at the top of the module, it is already being used in line 54.